ui/apps/platform/src/Containers/Vulnerabilities/utils/searchUtils.tsx

```diff
@@ -33,26 +32,6 @@ import {
 
 export type EntityTab = 'CVE' | 'Image' | 'Deployment';
 
-export type WorkloadCvesSearch = {
-    vulnerabilityState: VulnerabilityState;
-    entityTab?: EntityTab;
-    s?: SearchFilter;
-};
-
-export function parseWorkloadCvesOverviewSearchString(search: string): WorkloadCvesSearch {
```

Comment:
This function was unused, the remaining code was moved to a `utils/searchUtils.tsx` file specific to Workload CVEs.